katdal/ms_extra.py

```diff
@@ -42,6 +42,12 @@ def open_table(name, readonly=False, verbose=False, **kwargs):
     return tables.table(name, readonly=readonly, ack=verbose, **kwargs)
 
 
+def table_rows(name, readonly=True, verbose=False, **kwargs):
```

Comment:
I'm not a fan of a two-line function that is only ever used once (adding three more lines and an indirection to a different module for little gain). I would be happy to reinstate this once the second user surfaces.

There is probably a need to do further checks on the table, so then we will need it open in the calling code.